.github/workflows/static-test.yml

```diff
@@ -21,15 +25,17 @@ jobs:
         fetch-depth: 0
     - name: Setup git
       run: |
-        git fetch origin master:master_upstream --no-tags
+        if [ -n "${{ github.base_ref }}" ]; then
+          git fetch origin ${{ github.base_ref }}:${{ github.base_ref }} --no-tags
+        fi
         git config apply.whitespace nowarn
     - name: Fetch riot/riotbuild Docker image
       run: docker pull riot/riotbuild:latest
     - name: Run static-tests
-      run: |
-        docker run --rm                     \
-          -e CI_BASE_BRANCH=master_upstream \
-          -e GITHUB_RUN_ID=${GITHUB_RUN_ID} \
-          -v $(pwd):/data/riotbuild         \
-          riot/riotbuild:latest             \
+      run: |      
+        docker run --rm                             \
+          -e CI_BASE_BRANCH=${{ github.base_ref }}  \
```

Comment:
about my previous comment, your answer is still unclear because CI_BASE_BRANCH is set here anyway. Maybe `github.base_ref` is empty in the case of a branch ?

Reply:
Yepp. That is the case. See `github.base_ref` [here](https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#github-context).

Reply:
I added some comments to clarify.